code-examples/helm-kafka-example/README.md

```diff
@@ -33,10 +33,28 @@ $ oc apply -f ./rhoas-secrets.yaml
 $ oc apply -f ./rhoas-services.yaml
 ```
 
-5. Deploy the helm chart setting the appropriate values:
- 
-```
-$ helm install . --generate-name --set-string rhoas.config=<configmap-name>,rhoas.secret=<secret-name>
-```
+5. Deploy the helm chart with the appropriate values:
+
+`helm install` command is used to install a chart to the OpenShift cluster.
+`Values` is a built-in object that provides access to values passed in the chart.
+
+The Helm chart uses the follwing values in the templates:
+
+`rhoas.config` - Name of the ConfigMap object containing configurations deployed in the OpenShift cluster
+`rhoas.secret` - Name of the Secret object containing service-account credentials deployed in the OpenShift cluster
+
+
+
+There are various ways to provide values:
+
+    1. The default method is specifying values in the `values.yaml` file.
```

Comment:
A couple of comments:

- Make this a  bullet list (*) rather than a numbered list
- After the first item, maybe show an example of the values.yaml file you configured? 

So, putting that all together:

```
* The default method is specifying values in a YAML file called `values.yaml`. An example is shown below.

## Example values.yaml file
rhoas: 
   config: my-service-context-configuration
   secret: service-account-credentials
```